packages/styles/src/elements/reset.scss

```diff
@@ -0,0 +1,122 @@
+/* === Source: Josh Comeau, https://www.joshwcomeau.com/css/custom-css-reset/ (adapted) === */
+
+/*
+  Use a more-intuitive box-sizing model.
+*/
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+/*
+  Remove default margin
+*/
+* {
+  margin: 0;
+}
+
+/*
+  Typographic tweaks!
+  - Add accessible line-height
+  - Improve text rendering
+*/
+body {
+  line-height: 1.5;
```

Comment:
I think we can remove this, as we will define it anyway ;)
Everything else looks good!

Reply:
It is removed.